Graphics/GraphicsEngineWebGPU/src/BufferWebGPUImpl.cpp

```diff
@@ -199,88 +192,41 @@ WGPUBuffer BufferWebGPUImpl::GetWebGPUBuffer() const
     return m_pDevice->GetDynamicMemoryManager().GetWGPUBuffer();
 }
 
-void BufferWebGPUImpl::Map(MAP_TYPE MapType, MAP_FLAGS MapFlags, PVoid& pMappedData)
+Uint64 BufferWebGPUImpl::GetDynamicOffset(DeviceContextIndex CtxId, const DeviceContextWebGPUImpl* pCtx) const
```

Comment:
This function is kept in the header to help compiler inline it.
Do not mix changes that are not related to the topic of PR.